framework/src/org/checkerframework/framework/util/PurityAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,26 @@
+package org.checkerframework.framework.util;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.checkerframework.common.basetype.BaseAnnotatedTypeFactory;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.qual.Unqualified;
+
+public class PurityAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
+
+    public PurityAnnotatedTypeFactory(BaseTypeChecker checker) {
+        super(checker);
+        this.postInit();
+    }
+
+    @Override
+    protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
+        return Collections.unmodifiableSet(
+                new HashSet<Class<? extends Annotation>>(
+                        Arrays.asList(Unqualified.class)));
```

Comment:
This class, and framework/util/PurityChecker doesn't seem to be used at all. There's a separate PurityChecker in dataflow/util which is used by BaseTypeVisitor.

Should I remove this class and framework/util/PurityChecker?